.ci/containers/membership-checker/reviewer_assignment.go

```diff
@@ -0,0 +1,206 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func reviewerAssignment(author, prNumber, GITHUB_TOKEN string) error {
+	if isNoAssigneeUser(author) {
+		fmt.Println("User is on the list, not assigning")
+		return nil
+	}
+
+	requestedReviewer, err := getPullRequestRequestedReviewer(prNumber, GITHUB_TOKEN)
+	if err != nil {
+		return err
+	}
+
+	previousAssignedReviewers, err := getPullRequestPreviousAssignedReviewers(prNumber, GITHUB_TOKEN)
+	if err != nil {
+		return err
+	}
+
+	if requestedReviewer != "" {
```

Comment:
hmm, the [previous logic](https://github.com/GoogleCloudPlatform/magic-modules/blob/eb6d15f1358aa286b99634a38227d4e04da701e3/.ci/containers/contributor-checker/check-contributor.sh#L24) seems to have been:

- ASSIGNEE is first requested reviewer or first reviewer
- if there is an assignee (that is, at least one requested reviewer or reviewer), re-request reviews from all team reviewers and stop
- Otherwise, randomly assign someone

That makes sense - so, we do need something like requestedReviewer. I'll make a suggestion below, since I still think removing this section will be useful.